app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -406,6 +406,96 @@ class TopicLessonsFragmentTest {
     }
   }
 
+  @Test
```

Comment:
Please add tests related to content description also.
Similar to `testLessonsPlayFragment_loadRatiosTopic_completeStoryProgress_contentDescriptionIsCorrect`
like
`testLessonsPlayFragment_loadRatiosTopic_partialStoryProgress_contentDescriptionIsCorrect`

Reply:
done